hrpsys_ros_bridge/scripts/sensor_ros_bridge_connect.py

```diff
@@ -53,6 +53,11 @@ def initSensorRosBridgeConnection(url, simulator_name, rosbridge_name, managerho
         time.sleep(1);
         sh=rtm.findRTC('sh')
         print program_name, " wait for 'sh' : ",sh
+
+    while sh.isInactive(): ## wait for initalizing all components
+        time.sleep(1)
+        sh=rtm.findRTC('sh')
```

Comment:
Thank you. I think 
```
  while sh.isInactive(): ## wait for initalizing all components
        time.sleep(1)
```
is enough

Reply:
I thought so.
But, it did not work correct without ```sh=rtm.findRTC('sh')```
Because ```rtc.ref.get_participating_contexts()``` may change after activated.
https://github.com/fkanehiro/hrpsys-base/blob/master/python/rtm.py#L452-L454